lib/manager/nuget/extract.ts

```diff
@@ -112,6 +113,42 @@ export async function extractPackageFile(
     return { deps };
   }
 
+  if (packageFile.endsWith('global.json')) {
+    const deps: PackageDependency[] = [];
+    let manifest: MsbuildGlobalManifest;
+
+    try {
+      manifest = JSON.parse(content);
+    } catch (err) {
+      logger.debug({ fileName: packageFile }, 'Invalid JSON');
+      return null;
+    }
+
+    if (manifest['msbuild-sdks'] === undefined) {
```

Comment:
```suggestion
    if (!manifest['msbuild-sdks'] && !manifest.sdk) {
```

see other comment

Reply:
Summarizing the above suggestions, I used the following: `if (!manifest['msbuild-sdks'] && (!manifest.sdk || !manifest.sdk.version)) { }`.

Reply:
@viceice do you agree?

Reply:
See my updated suggestion

Reply:
@viceice really good suggestion! I'll make another commit, since I can't apply this because it is outdated.